media/videostitcher/stitcher_test.go

```diff
@@ -186,23 +200,82 @@ func TestCdnKeys(t *testing.T) {
 	}
 
 	// Tests
-	// Google CDN tests
+	// Media CDN tests
+
+	// Create a new Media CDN key.
+	testutil.Retry(t, 3, 2*time.Second, func(r *testutil.R) {
+		cdnKeyName := fmt.Sprintf("projects/%s/locations/%s/cdnKeys/%s", projectNumber, location, mediaCdnKeyID)
+		if err := createCdnKey(buf, tc.ProjectID, mediaCdnKeyID, hostname, keyName, mediaCdnPrivateKey, true); err != nil {
+			r.Errorf("createCdnKey (Media CDN) got err: %v", err)
```

Comment:
Here and elsewhere, consider using `r.Fatalf` if this operation needs to succeed in order for subsequent steps to work.

Reply:
Done